repositories.gradle

```diff
@@ -7,6 +7,7 @@ repositories {
     maven {
         name 'ic2'
         url 'https://maven.ic2.player.to/'
+        url = "https://maven2.ic2.player.to/"
```

Comment:
This change doesn't look relevant for this MR (and its inconsistent code style). Can you revert this?

Reply:
This is important, the main ic2 maven mirror is down and the second one should be added to all repos going forward to keep them possible to build

Reply:
But yeah, I don't think the `=` is required in Groovy syntax

Reply:
my IDEA goes mad, compilation works though
<img width="426" alt="2022-05-04 131129" src="https://user-images.githubusercontent.com/40035906/166622259-f431646a-d88f-441a-acfe-8a9467d240d9.png">